docker/nodejs/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM nginx:latest
+
+########### Nginx Configuration starts here ###########
```

Comment:
why is there an nginx configuration in nodejs? 

Shouldn't this be a separate container with nginx linking both node.js and django specific containers.